src/components/actions/SwapDetails.jsx

```diff
@@ -106,15 +107,15 @@ export default function SwapDetails() {
   };
 
   const effects = [
-    {
-      title: "Asset A Total",
-      val: `${assetA.type}: ${formatPrice((pool.state.totalPrimary/1000000).toFixed(0))}`,
-      hasToolTip: false,
-    },
-    {
-      title: "Asset B Total",
-      val: `${assetB.type}: ${formatPrice((pool.state.totalSecondary/1000000).toFixed(0))}`,
```

Comment:
Could we keep the old version in the comments? I made these changes to appease Ryan/Rylie